api/observability/v1/output_types.go

```diff
@@ -769,3 +774,50 @@ type Syslog struct {
 	// +kubebuilder:validation:Optional
 	MsgID string `json:"msgID,omitempty"`
 }
+
+type OtlpTuningSpec struct {
+	BaseOutputTuningSpec `json:",inline"`
+
+	// Compression causes data to be compressed before sending over the network.
+	// It is an error if the compression type is not supported by the output.
+	//
+	// +kubebuilder:validation:Enum:=gzip;none
+	// +kubebuilder:default:=gzip
+	Compression string `json:"compression,omitempty"`
+}
+
+// Otlp provided configuration for sending OTLP/HTTP with JSON-encoded Protobuf payload requests.
+// https://opentelemetry.io/docs/specs/otlp/#json-protobuf-encoding
+type Otlp struct {
+	// URL to send log records to.
```

Comment:
Based upon [this](https://github.com/openshift/enhancements/pull/1618#discussion_r1601613595) discussion this should state "must terminate".  Additionally, add [validation](https://book.kubebuilder.io/reference/markers/crd-validation) to enforce this change:

```
+kubebuilder:validation:Pattern
```
There are examples of usage elsewhere within the API, maybe for serviceAccount?